src/python/pants/notes/2.8.x.md

```diff
@@ -1,5 +1,53 @@
 # 2.8.x Release Series
 
+## 2.8.0.dev1 (Sep 10, 2021)
+
+### New Features
+
+* Support --fail-under on merged coverage. ([#12778](https://github.com/pantsbuild/pants/pull/12778))
+
+* Add python-docx to the module mapping dictionary ([#12775](https://github.com/pantsbuild/pants/pull/12775))
+
+* Add python-pptx to the module mapping dictionary ([#12776](https://github.com/pantsbuild/pants/pull/12776))
+
+* Add `opencv-python` to the default Python module mapping ([#12777](https://github.com/pantsbuild/pants/pull/12777))
+
+* Add `PyMuPDF` to the default Python module mapping ([#12774](https://github.com/pantsbuild/pants/pull/12774))
+
+### User API Changes
+
+* Deprecate `--list-provides` option. ([#12759](https://github.com/pantsbuild/pants/pull/12759))
+
+* Upgrade default `isort` to latest `isort==5.9.3` ([#12756](https://github.com/pantsbuild/pants/pull/12756))
+
+### Plugin API Changes
+
+* `OutputPathField.value_or_default()` no longer has an `Address` argument ([#12837](https://github.com/pantsbuild/pants/pull/12837))
+
+### Bug fixes
+
+* Properly include file dependencies in docker build context ([#12758](https://github.com/pantsbuild/pants/pull/12758))
+
+* DigestSubset should not short-circuit when there are ignores involved. ([#12648](https://github.com/pantsbuild/pants/pull/12648))
+
+### Performance
+
+* Improve cache reuse for `./pants package` when using a constraints file or lockfile ([#12807](https://github.com/pantsbuild/pants/pull/12807))
+
+* Upgrade to Pex 2.1.48 and leverage packed layout. ([#12808](https://github.com/pantsbuild/pants/pull/12808))
+
+* Fix backports of std lib modules like `dataclasses` not working with dependency inference ([#12818](https://github.com/pantsbuild/pants/pull/12818))
+
+### Documentation
+
+* Warn if `[python-repos]` is set during lockfile generation ([#12800](https://github.com/pantsbuild/pants/pull/12800))
+
+* Add `version` to lockfile metadata headers ([#12788](https://github.com/pantsbuild/pants/pull/12788))
```

Comment:
User api change

Reply:
How so? The version is part of the do not touch internal metadata. The rest of the change is adding better error messages. Its the latter I used to classify as documentation.

Reply:
I can totally alter the message though if that helps and only mention the improved lockfile error messages bit.

Reply:
Aha - 3 of the lockfiles got more than a version header, they actually got bumped. Ok.

Reply:
I was more thinking that you need to regenerate your lockfiles now to have the new metadata format